examples/11-http-blocking.php

```diff
@@ -1,10 +1,6 @@
 <?php
 
 use Clue\React\Block;
-use Clue\React\Mq\Queue;
-use Psr\Http\Message\ResponseInterface;
-use React\EventLoop\Factory;
-use React\Http\Browser;
```

Comment:
LGTM :+1: Just noticed the example still references the `$loop` variable, can you update this? :+1:

Reply:
It actually still needs to reference the `$loop` variable in this case, but there was a `use` statement missing.